src/routes/subscriptions.js

```diff
@@ -0,0 +1,71 @@
+const { StatusCodes } = require('http-status-codes');
+
+const express = require('express');
+const router = express.Router();
+
+// POST /
+// Recieve subscription notification
+router.post('/:id', (req, res) => {
+  const id = req.params.id;
+  res.sendStatus(StatusCodes.OK);
+  const planDef = getPlanDef(id);
+  usePlanDef(planDef);
+});
+
+// if the subscription returns a resource, the notification
+// will be a PUT instead of a POST
+router.put('/:id/:resource/:resourceId', (req, res) => {
+  const id = req.params.id;
+  res.sendStatus(StatusCodes.OK);
+  const planDef = getPlanDef(id);
+  usePlanDef(planDef, req.body);
+});
+
+function getPlanDef(id) {
```

Comment:
Is it too much effort to add the persistence here? Since the interface already exists it seems like it should be easy enough to set up a read function (with a manual write so we can load a PlanDefinition for testing). I don't want to hold this up though.